### ISSUE TITLE
SQLite support

### ISSUE THREAD

**Issue by DevDevious** [go-jet/jet#84]
@go-jet , issues #43 and #76 mention SQLite support coming soon. Are there any plans on making this happen in the foreseeable future?

**Comment by go-jet** [comment]
I believe it should be included in the next release.

**Comment by MagnifiGo** [comment]
Is there a date planned ?

**Comment by go-jet** [comment]
No, the date has not been set yet. Depending on the complexity, the next release may be in the next 1 to 3 months.
